packages/framework-integration-tests/integration/provider-unaware/load/artillery-executor.ts

```diff
@@ -0,0 +1,63 @@
+import * as fs from 'fs'
+import * as yaml from 'yaml'
+import * as path from 'path'
+import { ProviderTestHelper } from '@boostercloud/application-tester'
+import { runCommand } from '../../helper/run-command'
+import { CloudFormation } from 'aws-sdk'
+
+const cloudFormation = new CloudFormation()
+
+interface Phase {
+  duration: number
+  arrivalRate: number
+}
+
+interface OverrideOptions {
+  phases?: Array<Phase>
+  variables?: Record<string, any>
+}
+
+export class ArtilleryExecutor {
```

Comment:
This is a class that can be used from an integration (or better called load) test to execute any Artillery script using Serverless artillery, allowing to execute thousands of requests per second